### ISSUE TITLE
Update references to the new API in EF 9 with links to the API documentation

### ISSUE THREAD

**Issue by AndriySvyryd** [dotnet/EntityFramework.Docs#4837]
Depends on the API documentation being published first

**Comment by roji** [comment]
Add to our release checklist (post-release steps)? Also, we need to update the link to the API docs from the conceptual TOC as well.

**Comment by SamMonoRT** [comment]
This can be closed right? @AndriySvyryd

**Comment by AndriySvyryd** [comment]
No. Someone still needs to go through and add links to the API docs from:

- https://learn.microsoft.com/ef/core/what-is-new/ef-core-9.0/whatsnew
- https://learn.microsoft.com/ef/core/what-is-new/ef-core-9.0/breaking-changes
- https://learn.microsoft.com/ef/core/providers/cosmos/querying
- https://learn.microsoft.com/ef/core/modeling/data-seeding
